.github/workflows/check_template_files.yaml

```diff
@@ -5,13 +5,22 @@ on: push
 jobs:
   check-template-files:
     runs-on: ubuntu-latest
+
+    name: Check template files
+
     steps:
-      - uses: actions/checkout@v3
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v4
+      - name: Checkout repository
+        id: checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.12
+        id: setup-python
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.9
+          python-version: '3.12'
```

Comment:
This is not in the other workflows. Why is this needed here?

Reply:
Because we just setup Python here and do not execute the full commons action (which has a default Python version).

Reply:
Missed tht detail, thanks.
Then this should be mergeable